tests/functional/spec/components/radio/radio_open_answers.spec.js

```diff
@@ -0,0 +1,28 @@
+const RadioOpenTruePage = require('../../../generated_pages/radio_open/radio-open-true.page.js');
+const RadioOpenFalsePage = require('../../../generated_pages/radio_open/radio-open-false.page.js');
+const RadioOpenNonePage = require('../../../generated_pages/radio_open/radio-open-none.page.js');
+
+describe('Given I start a Radio survey with a write-in option', function() {
+  beforeEach(function() {
+    browser.openQuestionnaire('test_radio_open.json');
+    $(RadioOpenTruePage.coffee()).click();
+  });
+
+  it('When I view a write-in radio and the open option is set to true, detail answer label should be displayed', function() {
```

Comment:
The tests are mostly consistent with descriptions formatted as "When ..., Then ..." so we should do that here.

Reply:
Fixed. Thanks!